packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Okta Hosted Vs. Customer Hosted
+---
+
+## Okta Hosted Vs. Customer Hosted
+
+Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta-hosted Sign-In Widget and the Customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. The difference between the two concepts is where the Sign-In Widget itself is hosted.
+
+## Okta-hosted widget
+
+The Okta-hosted Sign-In Widget is considered the easiest and most secure means of integration. This is because the Sign-In Widget itself is hosted by Okta, maintained by Okta, and kept secure by Okta. The Okta-hosted Sign-In Widget is recommended for most integrations. The OpenID Connect redirect flow is used to redirect users to Okta, where the user authenticates and is redirected back to a specified, allow listed, redirect URI. This design is considered security best practice. The look and feel, as well as the enabled features, are all configured within the Okta Admin Console. HTML customization of the page is accessible in the Admin Console, which provides a moderate level of customization to the experience itself.
+
+## Customer-hosted widget
+
+The Customer-hosted widget is considered the best balance of flexibility and effort to integrate. The Customer-hosted widget is recommended if an integration requires a deeper level of customization or a Okta-hosted experience isn't possible. The Customer-hosted widget works by providing an open source [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) to embed into the page. The widget is built and updated by Okta, uses industry best practice security design, and is added to a page with a few lines of HTML/JavaScript. The widget itself can be loaded directly from the CDN, NPM, or built from source. The look and feel is customized directly through HTML/CSS/SASS and JavaScript. Features are configured within the Okta Admin Console and enabled in the widget through JavaScript.
```

Comment:
```suggestion
The Customer-hosted Sign-In Widget is considered the best balance of flexibility and effort to integrate. The Customer-hosted Sign-In Widget is recommended if an integration requires a deeper level of customization or a Okta-hosted experience isn't possible. The Customer-hosted Sign-In Widget works by providing an open source [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) to embed into the page. The Sign-In Widget is built and updated by Okta, uses industry best practice security design, and is added to a page with a few lines of HTML/JavaScript. It can be loaded directly from the CDN, NPM, or built from source. The look and feel is customized directly through HTML/CSS/SASS and JavaScript. Features are configured within the Okta Admin Console and enabled through JavaScript.
```